components/microsoft_onedrive/sources/new-folder/new-folder.mjs

```diff
@@ -0,0 +1,29 @@
+import base from "../new-file/new-file.mjs";
+
+const {
+  props,
+  methods,
+  hooks,
+  run,
+} = base;
+
+export default {
+  type: "source",
+  key: "microsoft_onedrive-new-folder",
+  name: "New Folder (Instant)",
+  description: "Emit new event when a new folder is created in a OneDrive drive",
+  version: "0.0.1",
```

Comment:
When this source emits an event, the summary says "New file". Could we overwrite `generateMeta` here so that it will say "New folder"? Or add a method `getSummary` to be called from `generateMeta `?